addons/mail/static/src/core/common/persona_model.js

```diff
@@ -144,6 +146,12 @@ export class Persona extends Record {
         return this.store.DEFAULT_AVATAR;
     }
 
+    get presenceChannel() {
+        return `odoo-presence-${this.type === "partner" ? "res.partner" : "mail.guest"}_${
+            this.id
+        }-${this.im_status_access_token}`;
```

Comment:
then have the token part as optional here

Reply:
It will correctly be parsed => empty string.